x11rb-protocol/Cargo.toml

```diff
@@ -18,6 +18,7 @@ serde = { version = "1", features = ["derive"], optional = true }
 
 [dev-dependencies]
 criterion = "0.3"
+fastrand = "1.7.0"
```

Comment:
Huh? Correct me if I am wrong, but.... unused dependency? (`git grep fastrand` only finds this line)

Reply:
I was working on a test framework for the generated code in another branch, and I accidentally added `fastrand` to this branch instead.